inc/saibfd.h

```diff
@@ -412,6 +429,15 @@ typedef enum _sai_bfd_session_attr_t
      */
     SAI_BFD_SESSION_ATTR_STATE,
 
+    /**
+     * @brief Offload type
+     *
+     * @type sai_bfd_session_offload_type_t
+     * @flags CREATE_ONLY
+     * @default SAI_BFD_SESSION_OFFLOAD_TYPE_NONE
+     */
+    SAI_BFD_SESSION_ATTR_BFD_SESSION_OFFLOAD_TYPE,
```

Comment:
maybe it should be name SAI_BFD_SESSION_ATTR_OFFLOAD_TYPE ? no need to duplicate bfd_session prefix

Reply:
Thanks. I renamed the Attribute as SAI_BFD_SESSION_ATTR_OFFLOAD_TYPE